Misc/NEWS.d/next/Library/2020-05-27-11-17-02.bpo-40784.hYEBjs.rst

```diff
@@ -0,0 +1 @@
+Fix :mod:`sqlite3` unit test for deterministic functions.
```

Comment:
A NEWS entry is only needed for user-facing bugs and features. This PR only fixes a test. I've just set 'skip news' label so you can safely delete it.

Reply:
It's fine to document fixed tests. But it may be good to move this NEWS entry into the Tests category, and maybe explain that it fix the test for SQLite 3.32.

I'm also fine with removing the NEWS entry ;-)

Reply:
Thanks! Noted for future reference :)